js/app.js

```diff
@@ -0,0 +1,100 @@
+//What is the difference between interpolation and concatenation?
+// Interpolation uses a variable to represent a value.
+//Concatenation combines two 'strings' together.
+
+//Give an example of each.
+// Interpolation:
+//// const verb = 'believe'
+// const noun = 'cheese'
+// console.log("Please " + verb + " the " + noun + ".")
+//
+//Concatenation:
+//const cityName = 'Astoria'
+//const weather = 'rainy'
+//console.log(cityName + " is " + weather + " this time of year.");
+//**ARE ALL CONCATENATIONS INTERPOLATIONS BUT NOT ALL INTERPOLATIONS ARE CONCATENATIONS??**
+
+//What does the acronym DRY stand for?
+//Don't Repeat Yourself
+
+//Why should we pay attention to it?
+//Helps utilize best practices in programming. DRY makes our code easier to read by other programmers
+//and helps us avoid bulky code and unessecary repetition.
+
+//What programming tools have we learned to write DRY code?
+//Thematically we discussed understaning the specific task at hand before rendering the code.
+//Specifially, we learned to use loops to avoid code repitition.
+
+//Section 2: Boolean Expressions
+const a = 4;
+const b = 53;
+const c = 57;
+const d = 16;
+const e = 'Kevin';
+
+console.log(a<b);
+console.log(c>d);
+console.log('Name'=== 'Name');
+console.log(a<b<c);
+console.log(a<a<d);
```

Comment:
same here -- leave some space for legibility (a < a < d)
but also... this isn't really right.  it turns out to be "true" but not for the reason you're expecting.

they wanted `a * a === d`